src/js/App.js

```diff
@@ -18,18 +18,21 @@
 
 import React, { Component } from 'react';
 import DataContainer from './dataLoader/DataContainer'
+import ThumbnailLoader from './dataLoader/Thumbnail'
```

Comment:
Would be more consistent to name the file ```ThumbnailLoader.js``` so it matches the class name. Easier to find etc.